action.yml

```diff
@@ -30,6 +30,10 @@ inputs:
     description: 'Version of Sphinx'
     required: false
     default: ''
+  SPHINXOPTS:
```

Comment:
Please use snake_case for this new input parameter. Besides, I favor sphinx_options instead of sphinx_opt.

We can declare the input parameter in lower case, but use upper case in main.sh, GitHub does this conversion for us.

Reply:
OK, I have changed to `sphinx_options`